pro-planner-client/src/index.js

```diff
@@ -18,7 +18,7 @@ import UserSelectionPage from './routes/UserSelectionPage';
 const router = createBrowserRouter([
 	{
 		path: '/',
-		element: <LandingPage />,
```

Comment:
Just to make this noted, this change makes sense for working on the branch, but shouldn't end up in main.

Reply:
Well noted.